src/iris/vendors/iris_twilio.py

```diff
@@ -20,15 +20,15 @@ class iris_twilio(object):
 
     def __init__(self, config):
         self.config = config
+        self.proxy_client = None
         if 'proxy' in self.config:
-            Connection.set_proxy_info(self.config['proxy']['host'],
-                                      self.config['proxy']['port'],
-                                      proxy_rdns=True)
+            self.proxy_client = TwilioHttpClient()
+            proxy = '%s:%s' % (self.config['proxy']['host'], self.config['proxy']['port'])
+            self.proxy_client.session.proxies = {'https': proxy, 'http': proxy}
         self.modes = {
             SMS_SUPPORT: self.send_sms,
             CALL_SUPPORT: self.send_call,
         }
-        self.timeout = config.get('timeout', 10)
```

Comment:
Is there a way of specifying the timeout with this new client?

Reply:
Thought of a better way to preserve timeouts. Updated.

Reply:
LGTM